03_letters/starfallz/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, strings.Join(sortLetters(letters("aba")), "\n"))
+}
+
+func letters(s string) map[rune]int {
+	letterCount := make(map[rune]int)
+
+	for _, currentLetter := range s {
+		letterCount[currentLetter]++
+	}
+
+	return letterCount
```

Comment:
Minor nitpick: on small functions like these, and between blocks that have just 1 line (the return),  internal line spacing doesn't add legibility, to the contrary. What do you think?